.goreleaser.yml

```diff
@@ -108,6 +108,12 @@ archives:
       - src: release_deps/observiq-otel-collector.service
         dst: "install"
         strip_parent: true
+      - src: release_deps/observiq-otel-collector
+        dst: "install"
+        strip_parent: true
+      - src: release_deps/sysconfig/observiq-otel-collector
+        dst: "install/sysconfig"
+        strip_parent: true
```

Comment:
I believe these can be sourced from `service/observiq-otel-collector` and `service/sysconfig/observiq-otel-collector`, and then you can revert the addition to `release-prep`. IIRC, `release-prep` is for generated configs.

Reply:
Okay, I was working from the systemd file as an example, and it is handled the same way. Is it being handled incorrectly?

Reply:
Its not wrong just more steps, we can keep doing it this way.

Reply:
Sounds good. I would like to try and maintain consistency with the systemd file wherever possible.